lib/faker/phone.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Faker.Phone do
+  @moduledoc """
+  Functions for generating phone numbers
+  """
+
+  @doc """
+  Return a random United States phone number
+  """
+  @spec us_phone(separator :: String.t) :: String.t
+  def us_phone(separator \\ "-") do
```

Comment:
I'm not sure about argument, it's better to provide random, cause user inputs can be random for example, as I understand `(333) 333-3333` is valid one and it won't fit this API. Lets stay with 0 arity.